### ISSUE TITLE
Legend focus indicator remains after losing focus

### ISSUE THREAD

**Issue by pvsleeper** [highcharts/highcharts#16633]
#### Expected behaviour
Focus indicator should disappear when focus is lost from legend when using the mouse and a11y is enabled.

#### Actual behaviour
The issue is slightly intermitted, but happens 9/10 times. The blue focus indicator border does not get removed from the legend when using the mouse to set focus on another element in the DOM. Using the keyboard however appears to be working fine.  

#### Live demo with steps to reproduce
https://jsfiddle.net/Ld09htsz/13/

#### Product version
9.3.1

#### Affected browser(s)
Google Chrome, Edge

**Comment by KacperMadej** [comment]
hi @pvsleeper 

Thank you for reporting the problem.

Not sure if the behavior is by design like that.

More specific a11y comments might follow soon.

**Comment by pvsleeper** [comment]
Hi @KacperMadej - thanks for looking into it. I feel pretty sure that the behavior is incorrect as it is a false focus that is being displayed which goes against the a11y rules

**Comment by oysteinmoseng** [comment]
Fixed in pending PR #16732. The fix should be part of the next official release.

Closing this issue manually as our automation won't catch it when multiple issues are fixed in one PR.
